wiki/versions/1.19/getting_started/setup/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: "org.quiltmc.loom"
```

Comment:
oh dear god, this example is cursed; I'd recommend going for something much closer to QTM, however, considering that this is only test code (at least that's what I hope), i can leave this one alone

Reply:
Eh, the projects are mostly for exposing the code in the pages, I don't think anyone is entering the repo then browsing the `build.gradle` to copy it. But yeah, a bit odd.
Removed the initializer and the properties, and only kept an empty build.gradle file to make the folder an actual subproject, as code is probably not needed for this section.